advocacy_docs/community/contributing/styleguide.mdx

```diff
@@ -594,3 +596,33 @@ When specifying dates for human readability, use the DD mmm YYYY format with a s
 
 When specifying dates as solely numbers, use [ISO8601](https://www.iso.org/iso-8601-date-and-time-format.html) format; YYYY/MM/DD. This is the internationally accepted, disambiguous format and should be used where you may expect the date to be read by automated systems.
 
+## Terminology considerations 
+
+### Ambiguity of the term "cluster"
+
+The term "cluster" can refer to different groups of resources in the technological field. 
+At EDB, the product documentation uses the term "cluster" to refer to not one, but several different objects/concepts.
+To avoid ambiguity, always use an adjective (describing word) to clarify which of the cluster types you are referring to in the specific context.
+
+Here are a few examples:
+
+**Postgres cluster**: Also known as a [database cluster](/epas/latest/fundamentals/epas_fundamentals/terminology/#database-cluster), a Postgres cluster is a Postgres server which hosts a number of databases in a single file system location.
+
+**PGD cluster**: A [PGD cluster](/pgd/latest/terminology/#pgd-cluster) is a [Postgres Distributed](/pgd/latest/) system with a group of multiple redundant database systems in different locations.
+
+**Kubernetes cluster**: A [Kubernetes cluster](https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-cluster) is a set of multiple nodes which run containerized applications.
+
+### Postgres distributions
+
+EDB products support three different types of Postgres distributions: 
+
+- PostgreSQL (open-source or community Postgres) 
+- EDB Postgres Advanced Server (EPAS) 
+- EDB Postgres Extended Server (PGE) 
+
+Although also referred to as "Postgres flavours", the preferred terms are "Postgres" or "Postgres distributions" to refer to all three options.
+
+## Trademark symbols 
+
+Use the appropriate ™ or ® symbol at least upon the first use of the trademark in the title and text. 
```

Comment:
Not in titles because it will make them unsearchable https://github.com/EnterpriseDB/docs/pull/5691